src/Sylius/Bundle/ChannelBundle/DependencyInjection/Compiler/CompositeRequestResolverPass.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ChannelBundle\DependencyInjection\Compiler;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+final class CompositeRequestResolverPass extends PrioritizedCompositeServicePass
+{
+    /**
+     * {@inheritdoc}
```

Comment:
I'm not convinced about that. For example in Product class in core bundle we use parent constructor, but don't use `{@inheritdoc}` tag.